user-service/mvn_dependency_tree.txt

```diff
@@ -272,20 +246,16 @@ org.aspectj/aspectjweaver/version = 1.9.4
 
 org.assertj/assertj-core/version = 3.11.1
 
-org.bouncycastle/bcpkix-jdk15on/version = 1.61
+org.bouncycastle/bcpkix-jdk15on/version = 1.60
 
-org.bouncycastle/bcprov-ext-jdk15on/version = 1.61
-
-org.bouncycastle/bcprov-jdk15on/version = 1.61
+org.bouncycastle/bcprov-jdk15on/version = 1.50
 
```

Comment:
This is quite a big step back in version for bouncycastle - can we fix at 1.61 ??